src/server/DeviceServer.js

```diff
@@ -87,9 +89,26 @@ class DeviceServer {
     this._eventPublisher = eventPublisher;
     this._areSystemFirmwareAutoupdatesEnabled =
       areSystemFirmwareAutoupdatesEnabled;
+
+    // Master listens to events from workers and broadcast them
+    // to all other workers except the owner.
+    if (useCluster && cluster.isMaster) {
```

Comment:
Yeah, I think the events should all be handled there -- I would think it already is handled by `process.on` but idk enough about clustering.